frontend/app/work_packages/directives/work-package-field-directive.js

```diff
@@ -50,20 +51,11 @@ module.exports = function(WorkPackageFieldService, EditableFieldsState) {
     };
 
     if (this.isEditable()) {
-      this.isBusy = false;
+      this.state.isBusy = false;
       this.isEditing = false;
       this.updateWriteValue();
       this.editTitle = I18n.t('js.inplace.button_edit', { attribute: this.field });
     }
-    
-    $scope.$watch('fieldController.isBusy', function(isBusy) {
-      EditableFieldsState.isBusy = isBusy;
-    });
-    $scope.$watch(function() {
-      return EditableFieldsState.isBusy;
-    }, function(isBusy) {
-      $scope.fieldController.isBusy = isBusy;
-    });
```

Comment:
Ah, here it is... okay... In my defense: Apparently I am not stupid, but simply blind...

Thanks for pointing out ^^